_posts/2024-06-10-first-cockpit-files-release.md

```diff
@@ -0,0 +1,27 @@
+---
+title: First Cockpit-Files release
+author: jelle
+date: 2024-06-10
+category: release
+tags: cockpit files manager browser
+---
+
+Last Friday we made the first release of cockpit-files, a new plugin for Cockpit to manage files on your server. Development on this new plugin was started by our GSoC student [Mahmoud Hamdy](https://github.com/MahmoudHamdy02) with the intention to replace existing [cockpit-navigator](https://github.com/45drives/cockpit-navigator/) plugin from 45Drives with a modern, automated tested and Cockpit styled plugin.
+
+This first release includes download, upload, copy & paste, renaming of files & directories, directory creation and editing of permissions. In future releases we intend to add a basic file editor for configuration files, cut & paste, SELinux, ACL's, symbolic link creation support and more.
+
+## Card view
+
+![Cockpit Files overview](/images/cockpit-files-1-overview.png)
+
+## List view
+
+![Cockpit Files list view](/images/cockpit-files-1-listview.png)
+
+## Upload support
+
+![Cockpit Files upload](/images/cockpit-files-1-upload.png)
+
+## Try it out
+
+We are working on getting Cockpit-files packaged in Fedora, Arch Linux and Debian. In the meantime this [copr repository](https://copr.fedorainfracloud.org/coprs/g/cockpit/main-builds/package/cockpit-files/) can be used to try out development snapshots, issues and feature requests can be reported on our [issue tracker](https://github.com/cockpit-project/cockpit-files/issues).
```

Comment:
@martinpitt taken the liberty to mention Debian but of course I won't be doing this, but will you? :)